web/beacon-app/src/features/projects/components/NewTopicModal.tsx

```diff
@@ -56,7 +56,6 @@ export const NewTopicModal = ({
             <Trans>New Topic</Trans>
           </h1>
         }
-        containerClassName="max-h-[90vh] overflow-scroll max-w-[80vw] lg:max-w-[40vw] no-scrollbar"
```

Comment:
This modal looks really wide on my screen. Should the max width remain?
![Screenshot 2023-07-26 at 12 31 30 PM (2)](https://github.com/rotationalio/ensign/assets/94616884/63cb2776-517e-4597-8d02-227e41751a2d)